### DISCUSSION TITLE
Using fortran code with IRAF?

### DISCUSSION THREAD

**Discussion by trappist1i** [iraf-community/iraf#376] [Q&A]
Hello,
Previously, turning a fortran executable into an iraf task required to save the executable to some directory, and to define it and mention its path in the login.cl file. But now that the login.cl file does not exist anymore, how could it be done?
Thanks!

**Comment by olebole** [comment]
Now there are several possible places for a `login.cl`:
* a global one for the machine, `/etc/iraf/login.cl`
* a user-specific one, `${HOME}/.iraf/login.cl`
* a local one, in the current directory

Depending on where you want your task available, you can use any of these: if the task should be available for all users, use `/etc/iraf/login.cl` (root/admin rights required), if you want to have it accessible for yourself, put it into `${HOME}/.iraf/login.cl`, and if you want to have it just in your current project, use a `login.cl` in your current directory.

The local default `login.cl` can be created with **mkiraf -c**, the user-specific one with **mkiraf -c -d**.
